src/webui/www/private/views/preferences.html

```diff
@@ -1860,6 +1873,19 @@
             $('port_value').setProperty('value', port);
         };
 
+        const generateRandomSslPort = function() {
```

Comment:
```suggestion
        const generateRandomSSLlPort = function() {
```
Please change all `Ssl` instances to `SSL`. I'm going omit mentioning this.

Reply:
Doing it, but you have other parts of code which work with SSL certificates which use the `.*Ssl.*` naming convention.

Reply:
I suppose they should be renamed at a later time...